docs/organization/dynamic-sampling/index.mdx

```diff
@@ -1,18 +1,54 @@
+
+![The dynamic sampling mode and target sample rate setting.](./img/mode-and-target-sample-rate.png)
+
+## Modes
+
+You can choose between two Dynamic Sampling modes for your organization: manual or automatic.
+
+    * **`automatic`** (default): In automatic mode, the sample rate for each project is set automatically based on relative project volume, so events fit within your organization’s billing quota. This boosts visibility into lower-volume projects. To account for any changes in event volume, sample rates are recalibrated on a regular basis. Automatic mode overrides any manually-set project sample rates. 
+    
+    * **`manual`**: If you want to set the exact sample rate for each project in your organization yourself, choose manual mode. Note, that in this mode, there is no automatic recalibration.
+
+
+[]: # (TODO: Rename AM3 with the actual user-facing name.)
+
 <Note>
 
-Retention priorities are only available on the latest version of our plans, starting from 1M reserved transactions. Anyone in your org with owner or billing-level permissions can update your plan by going to Settings > Subscription, and clicking the "Manage Subscription" button.
+Please note that these Dynamic Sampling modes are only available for customers on the AM3 platform.
 
 </Note>
 
-While storing all your data makes sense at relatively low volumes, as your application scales, storing a raw copy of all your data has diminishing returns. When data reaches high volumes, Sentry begins to automatically prioritize retaining certain transactions over others with server-side data retention strategies called retention priorities.
 
-It's important to note that even when we begin to only store the most valuable data based on retention priorities, the [performance metrics](/product/performance/metrics/) you see are still based on **all** the events you send to Sentry. This ensures that you get a full and complete view of your application's health.
+## Differences to Sampling in the Sentry SDKs
+
+Dynamic Sampling is different from the sampling that you can configure in the Sentry SDKs. The SDK sampling rate is applied before events are sent to Sentry, while Dynamic Sampling is applied after events are received by Sentry. This means that Dynamic Sampling can be used to adjust the sample rate for events that have already been sent to Sentry and only take into account the volume of events that have been received by Sentry.
+
+Also in contrast to the Dynamic Sampling, the SDK sample rate is never dynamically adjusted.
+
+## Dynamic Sampling Priorities
+
+<Note>
+
+Dynamic Sampling priorities are only available on the latest version of our plans, starting from 1M reserved transactions. Anyone in your org with owner or billing-level permissions can update your plan by going to Settings > Subscription, and clicking the "Manage Subscription" button.
```

Comment:
This will need to be updated. DS priorities are available on AM2 _and_ AM3 (right now this is still correct, as AM2 is latest).